docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -108,11 +108,12 @@ Use this transformation to add a new field calculated from two other fields. Eac
 
 - **Mode -** Select a mode:
   - **Reduce row -** Apply selected calculation on each row of selected fields independently.
-  - **Binary option -** Apply basic math operation(sum, multiply, etc) on values in a single row from two selected fields.
+  - **Binary operation -** Apply basic binary operations(sum, multiply, etc) on values in a single row from two selected fields.
```

Comment:
This is maybe a bit beyond the context of this PR but should it seems like we should include language about what these operators are. I'm assuming plenty of folks will know, but I'm guessing there are those that may be confused. What do you think?

Reply:
I agree! I've added explanations to each of the new functions added. I think binary operations should be fine as they are since they are pretty self explanatory. @imatwawana can I please ask for another review of the docs with these new modifications? Thank you!